src/Generator.php

```diff
@@ -234,10 +234,10 @@ private function allocateLocaleArray($path, $multiLocales = false)
                     continue;
                 }
                 if ($lastLocale !== false) {
-                    $root = realpath(base_path() . $this->config['langPath'] . '/' . $lastLocale);
-                    $filePath = $this->removeExtension(str_replace('\\', '_', ltrim(str_replace($root, '', realpath($fileName)), '\\')));
+                    $root = realpath(base_path() . $this->config['langPath'] . DIRECTORY_SEPARATOR . $lastLocale);
+                    $filePath = $this->removeExtension(str_replace(DIRECTORY_SEPARATOR, '_', ltrim(str_replace($root, '', realpath($fileName)), DIRECTORY_SEPARATOR)));
                     if ($multiLocales) {
-                        $this->filesToCreate[$lastLocale][$lastLocale][substr($filePath, 1)] = $this->adjustArray($tmp);
+                        $this->filesToCreate[$lastLocale][$lastLocale][$filePath] = $this->adjustArray($tmp);
```

Comment:
Also not sure about this change

Reply:
We no longer need to remove the leading '/' since it should be already removed at this point.